test-requirements.txt

```diff
@@ -7,3 +7,4 @@ mock==4.0.2; python_version >= '3.6'
 distro; python_version >= '3.8'
 nose
 nose-timer; python_version >= '2.7'
+enum34; python_version < '3.4'
```

Comment:
where is this being used?

Reply:
`tests.ga.extension_emulator.ExtensionCommandName` inherits from `enum.Enum`.

Reply:
+1 to @pgombar's comment. If its not adding too much extra benefit, might as well skip it

Reply:
I was at one point using the enum features to iterate over the enum values, something that is hacky at best without an enum, and that I figured would be worth a test import.

However, I'm pretty sure I'm not actually doing that anymore; I'm going to try reworking this to not have the import, and, if it works, will have no argument for keeping this!